CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Dates are in `dd-mm-yyyy` format.
 
+## [2.5.0] - XX-XX-2025
+
+### Added
+
+
+### Changed
+
+- Migration to NBGL API for Nano X and Nano S Plus devices including the ticker position change to after amount
```

Comment:
I only try to put changes that are relevant for users or developers, so they might not care about BAGL vs NBGL. Since there are many UX changes, I'd keep it generic with something like
```
- UX rehaul on Nano S Plus and Nano X devices.
```